src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/IsInRangeConverter.shared.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	public class IsInRangeConverter : BindableObject, IValueConverter
+	{
+		public bool RevertResult { get; set; } = false;
```

Comment:
Personally, I am against of "RevertResult" property approach.
Let's use IsNotInRangeConverter approach

CC @pictos @jfversluis

Reply:
Agreed

Reply:
Done